src/terraform/commands.go

```diff
@@ -10,6 +10,9 @@ import (
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	if !util.DoesFileExist(deployConfig.TerraformFile) {
+		return errors.New("terraform initialization attempted in directory with no 'main.tf' file. Please make sure 'exo deploy build' has been run")
+	}
```

Comment:
The only problem with that is automatically running `exo deploy build` will write a new `main.tf` file. One of the points of breaking this up into `exo deploy build` and `exo deploy` is that after running the first command, the user can inspect the `main.tf` file and make or add changes, and then deploy with those modifications.

However, overwriting the `main.tf` file is a problem we will have to deal with at some point. I was thinking maybe `exo deploy init` could be passed a flag to tell it whether or not to generate a new `main.tf` file. We could even have it generate a new `main.tf` file just with a different name. @alexdavid thoughts?